.circleci/config.yml

```diff
@@ -0,0 +1,84 @@
+version: 2
+jobs:
+  python-3.4:
+    working_directory: ~/coala
+
+    docker:
+      - image: circleci/python:3.4
+
+    steps: &build
+      - checkout
+      - restore_cache:
+          key: dependency-cache-{{ checksum "requirements.txt" }}
+
+      - run:
+          name: virtual-env
+          command: |
+            if [ ! -d "$HOME/.coala-venv" ]; then python3 -m venv ~/.coala-venv; fi
```

Comment:
why do we need a venv?

why not cache the pip download directory instead?

Reply:
how does it make it easier?  I am a technical person.  Dont spare me the details.